src/proxy-ui-api/frontend/src/components/ui/Snackbar.vue

```diff
@@ -26,38 +26,40 @@
 <template>
   <div>
     <!-- Success -->
-    <v-snackbar
-      data-test="success-snackbar"
-      v-for="notification in successNotifications"
-      :key="notification.timeAdded"
-      :timeout="notification.timeout"
-      v-model="notification.show"
-      :color="colors.Success10"
-      multi-line
-      class="success-snackbar"
-      :min-width="760"
-      @input="closeSuccess(notification.timeAdded)"
-    >
-      <div class="row-wrapper-top scrollable identifier-wrap">
-        <v-icon color="#0CC177">icon-Checker</v-icon>
-        <div class="row-wrapper">
-          <div v-if="notification.successMessageCode">
-            {{ $t(notification.successMessageCode) }}
-          </div>
-          <div v-if="notification.successMessageRaw">
-            {{ notification.successMessageRaw }}
+    <transition name="fade">
+      <v-snackbar
+        data-test="success-snackbar"
+        v-for="notification in successNotifications"
+        :key="notification.timeAdded"
+        :timeout="notification.timeout"
+        v-model="notification.show"
+        :color="colors.Success10"
+        multi-line
+        class="success-snackbar"
+        :min-width="760"
+        @input="closeSuccess(notification.timeAdded)"
+      >
+        <div class="row-wrapper-top scrollable identifier-wrap">
+          <v-icon color="#0CC177">icon-Checker</v-icon>
```

Comment:
Color code here is a magicvalue. Prefer using a variable.

Reply:
Fixed